src/components/portrait/portrait.style.js

```diff
@@ -7,6 +7,12 @@ import BaseTheme from "../../style/themes/base";
 import Icon from "../icon";
 import OptionsHelper from "../../utils/helpers/options-helper";
 
+function stylingForCursor({ onClick }) {
+  return css`
+    cursor: ${onClick ? "pointer" : "auto"};
+  `;
+}
+
```

Comment:
`auto` is the default so there's no need to specific that here, instead this should be 

```js
function stylingForCursor({ onClick }) {
  if (onClick) {
    return "cursor: pointer;";
  }

  return "";
}
```

Or something along those lines. You can see `stylingForSize` and `stylingForShape` doing something similar before (there's no need for the `css''` though, it can just be a string

Reply:
fixed